src/app/sign-up-metamask/sign-up-metamask.component.html

```diff
@@ -0,0 +1,98 @@
+<app-banner></app-banner>
+
+<div
+  class="container home-container text-center"
+  *ngIf="globalVars.inTab || globalVars.webview"
+>
+  <div *ngIf="this.currentScreen === SCREEN.CREATE_ACCOUNT">
+    <div class="mb-20px">
+      <span class="title-text">Sign up / Log in using MetaMask</span>
+      <div class="main-text mb-2">
+        Enter the Decentralized Social using your MetaMask wallet.
+      </div>
+      <p class="txtalj mt-20px">
+        DeSo is a novel blockchain built for web3 social networks. It enables a
+        new category of apps that combine the innovations of decentralized
+        finance and social. To get started, let's set you up with an account.
+        This Metamask integration will allow you to create a DeSo wallet derived
+        off of your ETH wallet. When you click the button below, you will be
+        asked to
+        <a
+          href="https://docs.metamask.io/guide/signing-data.html#signing-data-with-metamask"
+          >sign</a
+        >
+        a message that will tell the DeSo blockchain validators that you're
+        creating a new wallet
+      </p>
+      <button
+        class="button button-large mb-20px bg-dark text-white mt-20px"
+        (click)="launchMetamask()"
+      >
+        <img
+          *ngIf="metamaskState !== METAMASK.CONNECT"
+          src="assets/metamask.png"
+          height="32px"
+          width="32px"
+          class="mr-10px"
+        />
+
+        <div
+          class="spinner-border mr-3"
+          style="width: 25px; height: 25px"
+          *ngIf="metamaskState === METAMASK.CONNECT"
+        ></div>
+        <span class="font-weight-bold"
+          >Connect<span *ngIf="metamaskState === METAMASK.CONNECT"
+            >ing</span
+          ></span
+        >
+      </button>
+
+      <div *ngIf="metamaskState === METAMASK.ERROR">
+        <div class="text-danger">{{ errorMessage }}</div>
+      </div>
+    </div>
+  </div>
+  <div *ngIf="this.currentScreen === SCREEN.ACCOUNT_SUCCESS">
```

Comment:
```suggestion
  <div *ngIf="currentScreen === SCREEN.ACCOUNT_SUCCESS">
```
don't need this in your html files.